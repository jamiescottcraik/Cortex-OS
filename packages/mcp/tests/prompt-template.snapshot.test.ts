import { readFileSync } from 'node:fs';
import { resolve } from 'node:path';
import { expect, test } from 'vitest';

/** Snapshot test for web MCP interface prompt examples. */
test('web-mcp-interface examples remain stable', () => {
  const html = readFileSync(
    resolve(__dirname, '../src/web-mcp-interface.ts'),
    'utf8',
  );
  const match = html.match(/<div class="example">([\s\S]*?)<\/div>/);
  expect(match?.[1].trim()).toMatchSnapshot();
});
