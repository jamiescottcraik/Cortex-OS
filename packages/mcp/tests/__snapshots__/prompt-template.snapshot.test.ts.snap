// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`web-mcp-interface examples remain stable 1`] = `
<strong>Examples:</strong><br>
            • cortex mcp add --transport http Ref "https://api.ref.tools/mcp?apiKey=ref-e672788111c76ba32bc1"<br>
            • claude mcp add --transport http ref-server https://api.ref.tools/mcp --header "Authorization: Bearer token"<br>
            • gemini mcp add ref-server --url https://api.ref.tools/mcp --key ref-e672788111c76ba32bc1
`;
